src/MonoTorrent.Tests/Common/TorrentTest.cs

```diff
@@ -92,14 +92,12 @@ private BEncodedDictionary CreateInfoDict ()
         private BEncodedList CreateFiles ()
         {
             BEncodedList files = new BEncodedList ();
-            BEncodedDictionary file;
-            BEncodedList path;
 
-            path = new BEncodedList {
+            BEncodedList path = new BEncodedList {
```

Comment:
@ilyalatt Most modern style guides would recommend constructs like this be declared as a  `var` . Can resharper make this 'obvious' things var instead of using the full type?

Reply:
There are cases when a parameter name equals to field name so `this.` can not be removed. You can convert private field names from camelCase to _camelCase format in resharper.